In [1]:
    
def oops():
    raise IndexError
def catch():
    try:
        oops()
    except IndexError:
        print ('Caught Index Error')
In [ ]:
